qemu/qemu_test.go

```diff
@@ -145,7 +144,32 @@ func TestAppendDeviceNetwork(t *testing.T) {
 	testAppend(netdev, deviceNetworkString, t)
 }
 
-var deviceNetworkPCIString = "-device driver=virtio-net-pci,disable-modern=true,netdev=tap0,mac=01:02:de:ad:be:ef,bus=/pci-bus/pcie.0,addr=ff -netdev tap,id=tap0,ifname=ceth0,downscript=no,script=no,fds=3:4,vhost=on"
+var deviceNetworkStringMq = "-netdev tap,id=tap0,vhost=on,fds=3:4 -device driver=virtio-net-pci,netdev=tap0,mac=01:02:de:ad:be:ef,disable-modern=true,mq=on,vectors=6"
+
+func TestAppendDeviceNetworkMq(t *testing.T) {
+	foo, _ := ioutil.TempFile(os.TempDir(), "qemu-ciao-test")
```

Comment:
If I understand correctly, the QEMU package doesn't close any file descriptors.  This is the responsibility of the caller.  Is this correct?  If so we should close the FDs in this test ( and others ).  Currently, they're leaked, i.e., we're missing a foo.Close() and a bar.Close()

Reply:
@markdryan good catch. This was existing code which I missed. Let me fix this.